files/Aboot/boot0.j2

```diff
@@ -395,6 +395,9 @@ write_boot_configs() {
         fi
     fi
 
+    # setting secure_boot_enable=true when secure boot enabled
+    [ -f /bin/securebootctl ] && securebootctl secureboot -display | grep -i "Secure Boot enable" -q && echo "secure_boot_enable=y" >> /tmp/append
```

Comment:
Comment and code does not match, is it supposed to be "secure_boot_enable=true" or "secure_boot_enable=y"?